src/main/java/matjojo/mixin/villagerInitMixin.java

```diff
@@ -15,6 +19,8 @@
 
 @Mixin(VillagerEntity.class)
 public abstract class villagerInitMixin extends MerchantEntity {
+	
+	public static final Tag<Item> EMERALD_ITEMS = TagFactory.ITEM.create(new Identifier("c", "emerald_items"));
```

Comment:
You may want to change the namespace from `c` to `villagersfollowemeralds` to prevent conflicts with other mods that use short namespaces, like this:
```suggestion
	public static final Tag<Item> EMERALD_ITEMS = TagFactory.ITEM.create(new Identifier("villagersfollowemeralds", "emerald_items"));
```

When you do so, also rename the tag JSON file:
Old name: `src/main/resources/data/c/tags/items/emerald_items.json`
New name: `src/main/resources/data/villagersfollowemeralds/tags/items/emerald_items.json`

Reply:
Thanks. I had thought to do this, however, the tutorial I had read on tag creation told me to use the c for "common", the fabric tutorials are, however, out of date.

Reply:
As you renamed the name space, can you please also rename the file
`src/main/resources/data/c/tags/items/emerald_items.json`
to `src/main/resources/data/villagersfollowemeralds/tags/items/emerald_items.json`?

This tutorial from Github Docs may be useful: https://docs.github.com/en/repositories/working-with-files/managing-files/renaming-a-file

Reply:
Ah, alright. I hope I renamed the file correctly.